pkg/stub/handler.go

```diff
@@ -97,8 +112,27 @@ func (h *Handler) provisionServices(wt *v1alpha1.Walkthrough) (*v1alpha1.Walkthr
 	return wtCopy, nil
 }
 
-func (h *Handler) userRoleBindings(wt *v1alpha1.Walkthrough) (*v1alpha1.Walkthrough, error) {
-	wtCopy := wt.DeepCopy()
-	//ToDo Implement me
-	return wtCopy, nil
+func newRoleBinding(username, namsepace, roleName string) *rbacv1.RoleBinding {
+	return &rbacv1.RoleBinding{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "RoleBinding",
+			APIVersion: "rbac.authorization.k8s.io/v1",
```

Comment:
might be worth adding a label so we know it is our secret, that way we can easily find it again in the future if needed

Reply:
Labels are added to both namspaces and role bindings created by this operator here https://github.com/integr8ly/walkthrough-operator/pull/3/commits/dfab48c0961706050cebab0e874ad180e56c2df0